chrome_manifest_v3/class_app.js

```
@@ -0,0 +1,848 @@
+    return ta.innerHTML;
+    // jQuery way, less safe: return $("<textarea />").text(s).html();
+  }
+
+  /**
+   * Decode entities
+   */
+  decodeEntities(s) {
+    const self = this;
+    const dict_k = { '...': '&hellip;', '*': '&bullet;', '-': '&mdash;' };
+    let re, new_s;
+    g2t_each(dict_k, function (value, key) {
```

Biome (1.9.4)

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)